java/client/src/org/openqa/selenium/SearchContext.java

```diff
@@ -19,7 +19,7 @@
 
 import java.util.List;
 
-public interface SearchContext {
+public interface SearchContext{
```

Comment:
I was trying to add I parameter. But it was bad idea because end user will face "rawtype" warning  at their current projects.